host/platform/Windows/capture/NVFBC/src/nvfbc.c

```diff
@@ -383,9 +384,10 @@ static int pointerThread(void * unused)
       this->mouseVisible = pointer.visible;
       this->mouseHotX    = pointer.hx;
       this->mouseHotY    = pointer.hy;
+      hotspotUpdated     = true;
     }
 
-    if (events[0])
+    if (events[0] || hotspotUpdated)
```

Comment:
If a mouse move has never occurred since the host was started, the x/y position of the cursor will be invalid, this needs to be checked also.

Reply:
Done.